vmdb/app/assets/javascripts/cfme_application.js

```diff
@@ -710,8 +710,8 @@ function miqCheckMaxLength(obj){
   var counter=obj.getAttribute ? obj.getAttribute("counter") : "";
   if (obj.getAttribute && obj.value.length>ml) obj.value=obj.value.substring(0,ml);
   if (counter != "")
-    if (miq_browser != 'Explorer') $(counter).textContent = obj.value.length;
-    else $(counter).innerText = obj.value.length;
+    if (miq_browser != 'Explorer') $j('#' + counter).textContent = obj.value.length;
```

Comment:
@h-kataria: `$j('#' + counter).textContent` should be `$j('#' + counter)[0].textContent` . . . probably in other places as well.